BUILD.md

```diff
@@ -93,7 +92,7 @@ Optional:
 ```
   -DBuild_Studio=[OFF|ON]             default: OFF
   -DBuild_View2=[OFF|ON]              default: OFF
-  -DBuild_Post=[OFF|ON]               default: ON
+  -DBuild_Post=[OFF|ON]               default: OFF
   -DCMAKE_INSTALL_PREFIX=<path>       default: ./install
```

Comment:
Is ./install the default for shapeworks or the dependencies?

Reply:
For shapeworks, in CMakeLists.txt, we have:

```
set(CMAKE_INSTALL_PREFIX "${CMAKE_CURRENT_BINARY_DIR}/install" CACHE STRING "Install path prefix, \
prepended onto install directories.")
```